api/base/settings/defaults.py

```diff
@@ -189,3 +189,6 @@
 ENABLE_ESI = osf_settings.ENABLE_ESI
 VARNISH_SERVERS = osf_settings.VARNISH_SERVERS
 ESI_MEDIA_TYPES = osf_settings.ESI_MEDIA_TYPES
+
+ADDONS_FOLDER_CONFIGURABLE = ['box', 'dropbox', 's3', 'googledrive']
```

Comment:
No. However, this will go away when the minor refactors necessary to enable this for other addons go in. The alternative would be either doing all of those in this PR, or adding a flag to the models. Neither option is particularly appealing.